internal/pkg/aws/ecs/ecs.go

```diff
@@ -168,6 +181,58 @@ func (e *ECS) ServiceTasks(clusterName, serviceName string) ([]*Task, error) {
 	return tasks, nil
 }
 
+// DefaultCluster returns the default clusters in the account and region.
+func (e *ECS) DefaultCluster() (string, error) {
+	resp, err := e.client.DescribeClusters(&ecs.DescribeClustersInput{})
+	if err != nil {
+		return "", fmt.Errorf("get default cluster: %w", err)
+	}
+
+	if len(resp.Clusters) == 0 {
+		return "", errors.New("no default cluster is found")
```

Comment:
Can you make this error a const at the top? This will let us compare errors in the caller.

Like I can imagine us doing something like:

```golang

 cluster, err := ecs.DefaultCluster()
 if err == ecs.NoDefaultClusterErr {
    cluster = ecs.CreateDefaultCluster()
  }
```